src/components/dashboard/Dashboard.js

```diff
@@ -260,16 +215,10 @@ const Dashboard = props => {
     if (!params) {
       return
     }
-    const { paymentCode, event } = params
 
-    if (paymentCode) {
-      //payment link (from send)
-      handleWithdraw(params)
-    } else if (event) {
+    const { event } = params
```

Comment:
maybe reconsider renaming handleAppLinks as this method was moved HandlePaymentLinks